### PR TITLE
Added map ability + slight color coding. 

### PR DESCRIPTION
**Pull request by ArchdukeTim**
Added plotting points within 20 'steps' (not entirely sure what the walk list is).
Pokemon are color coordinated to help differentiate, but there's probably a better way to get them to look less similar, especially since many Pokemon are below 40.

Right now it just uses a multiplier and the Pokemon ID. Maybe using the name of the pokemon to generate a RGB would be more random with closer together pokemon?

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -116,7 +120,7 @@ def api_req(api_endpoint, access_token, *mehs, **kw):
             print("\n\n")
 
         print("Sleeping for 2 seconds to get around rate-limit.")
-        time.sleep(2)
+        time.sleep(1)
```
Comment (ArchdukeTim) [inline main.py:123]: I also found that this doesn't cause too many issues. None that are there with it at 2.

File: main.py [hunk 2]
```diff
@@ -49,6 +51,8 @@ def getNeighbors():
 FLOAT_LAT = 0
 FLOAT_LONG = 0
 
+mymap = pygmaps.pygmaps(38.8855018,-77.1796793,15.08)
```
Comment (marauder37) [inline main.py]: Could you get these co-ordinates from the origin instead of hard-coding your location?

**Comment by MonsterMMORPG** [comment]
Someone can please make a tutorial how can we locally merge the pull requests to use the latest version?

**Comment by TheZ3ro** [comment]
@Twinters007 can you take a look at this Pull Request https://github.com/leegao/pokemongo-api-demo/pull/4

We print on the map the Pokemon Sprite and we added the ability to hide useless pokemon.

If you want you can merge your changes with ours ;)

**Comment by marauder37** [comment]
@MonsterMMORPG 
1. be in your local clone of the repository: `cd ~/Documents/Pokemon/pokemongo-api-demo`
1a. be in the correct branch of your repository: `git checkout simulation`
2. add twinters's repository as a remote: `git remote add twinters https://github.com/Twinters007/pokemongo-api-demo.git`
3. update your list of available remote branches: `git fetch --all`
4. see what will happen if you do the merge: `git diff twinters/simulation`
5. do the merge: `git merge twinters/simulation`

This will give you all of twinters's changes, not just the 4 commits on this PR, but it'll probably be fine.

If you're fussy, instead of step 5 merge the individual commits: `git cherry-pick 06436d2` etc

**Comment by ArchdukeTim** [comment]
@TheZ3ro  What would you need merged? I'm about to take a look but if yours does autoupdate there's nothing I have that you don't.

**Comment by TheZ3ro** [comment]
@Twinters007 I don't know, maybe you have some different idea for the map implementation  :wink:

**Comment by MonsterMMORPG** [comment]
@Twinters007 can you check whether the searching algorithm can be improved or not? i am running 3 instances and nothing worthy appears in my city :(

**Pull request closed without merging**